operator/config/metalk8s/delete_ns.yaml

```diff
@@ -0,0 +1,5 @@
+$patch: delete
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: system
```

Comment:
From your commit message, I understand why we don't want to apply the `kube-system` *Namespace* manifest, but what about this one? Did we create it before? Or is it just part of the "default config" which we include and translated to `kube-system` somehow?

Reply:
> is it just part of the "default config" which we include and translated to kube-system somehow?

Yes, by default with operator SDK we have this manifest 
```yaml
apiVersion: v1
kind: Namespace
metadata:
  labels:
    control-plane: controller-manager
  name: system
```

That we update with kustomize, but to remove it we need to refer to it using the "original" name